code/datums/diseases/kingstons.dm

```diff
@@ -0,0 +1,46 @@
+/datum/disease/kingstons
+	name = "Kingstons"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Milk."
```

Comment:
Why does this one have a period/fullstop and the other does? Determine which is needed, and correct the incorrect one.